source/Applications/Advanced/CreateDepthMap/CreateDepthMap.cpp

```diff
@@ -15,32 +15,26 @@ This example shows how to convert point cloud from ZDF file to OpenCV format, th
 
 namespace
 {
-    enum class Axis
-    {
-        x,
-        y,
-        z
-    };
-
-    template<Axis axis>
-    float getValue(const Zivid::PointXYZ &p);
-
-    template<>
-    float getValue<Axis::z>(const Zivid::PointXYZ &p)
+    float getValue(const Zivid::PointZ &p)
```

Comment:
This is now specific to Z so I would call it getValueZ

Reply:
done